scripts/bundlingUtils.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs');
+const path = require('path');
+
+// Function to update package.json
+function updatePackageJson() {
+  const packagePath = './package.json';
+
+  fs.readFile(packagePath, 'utf8', (err, data) => {
+    if (err) {
+      console.error(`Error reading package.json: ${err}`);
+      return;
+    }
+
+    try {
+      const packageJson = JSON.parse(data);
+
+      // Update package name if necessary
+      if (packageJson.name && packageJson.name === '@salesforce/core') {
+        packageJson.name = '@salesforce/core-bundle';
+      }
+
+      // Remove 'prepack' and 'prepare' scripts
+      if (packageJson.scripts) {
+        delete packageJson.scripts.prepack;
+        delete packageJson.scripts.prepare;
+      }
+
+      // Remove 'exports'
+      if (packageJson.exports) {
+        delete packageJson.exports;
+      }
+
+      fs.writeFile(packagePath, JSON.stringify(packageJson, null, 2), 'utf8', (writeErr) => {
+        if (writeErr) {
+          console.error(`Error writing to package.json: ${writeErr}`);
+        } else {
+          console.log('package.json updated successfully.');
+        }
+      });
+    } catch (parseErr) {
+      console.error(`Error parsing JSON in package.json: ${parseErr}`);
+    }
+  });
+}
+
+// Function to check the path to transformStream
+function checkTransformStreamPath() {
+  const loggerPath = './src/logger/logger.ts';
+  const targetString = "target: path.join('..', '..', 'lib', 'logger', 'transformStream')";
+  const replacementString = "target: './transformStream'";
+
+  fs.readFile(loggerPath, 'utf8', (err, data) => {
+    if (err) {
+      console.error(`Error reading logger.ts: ${err}`);
+      return;
+    }
+
+    // Check if the target string exists in the file
+    if (!data.includes(targetString)) {
+      console.error(
+        `Error: The target string "${targetString}" was not found in logger.ts.\n Please make sure to bundle transformStream by referencing the new path or reach out to IDEx Foundations Team.`
+      );
+      return;
+    }
+  });
+}
+
+// Function to update logger.ts
+function updateLoggerTs() {
+  const loggerPath = './src/logger/logger.ts';
```

Comment:
can this and `checkTransformStreamPath` be combined? they're doing a lot of the same? at least share some of the variables